### PR TITLE
zooooooooooooom

### PR DESCRIPTION
**Pull request by rschifflin**
@acmcarther with z and x. Looks like the eye is actually fixated on the topright corner instead of screen center when using `look_at` however, which is strange

### CODE REVIEW

File: prototype1/src/client/mod.rs [hunk 1]
```diff
@@ -149,7 +149,10 @@ mod client {
     );
 
     let proj = cgmath::perspective(cgmath::deg(45.0f32),
-                                   stream.get_aspect_ratio(), 1.0, 40.0);
+                                   stream.get_aspect_ratio(), 1.0, 400.0);
```
Comment (acmcarther) [inline prototype1/src/client/mod.rs:152]: why did you change the perspective projection?

File: prototype1/src/client/mod.rs [hunk 2]
```diff
@@ -261,19 +270,19 @@ mod client {
...
           color: [0.3, 0.3, 0.3, 1.0],
           depth: 1.0,
           stencil: 0,
       });
-      client_state.entities.iter().foreach( |(id, primitive): (&u8, &Primitive)| {
-        println!("ents {} {:?}", id, primitive);
+      client_state.entities.iter()
+        .chain(vec![origin_entity].into_iter())
+        .foreach( |(_, primitive): (&u8, &Primitive)| {
         let texture = factory.create_texture_rgba8(1, 1).unwrap();
```
Comment (acmcarther) [inline prototype1/src/client/mod.rs]: Please scoot this stuff in for this block

**Comment by acmcarther** [comment]
I'm going to check it out locally, but you shouldn't have had to do that change to the P matrix. The scalar multiplication of the view matrix should have been sufficient.

**Comment by rschifflin** [comment]
I just changed the far clipping plane so I could zoom waaay out

File: prototype1/src/client/mod.rs (hunk 1 above)
Comment (acmcarther) [inline prototype1/src/client/mod.rs:152]: o i see, this is the draw distance.

**Comment by acmcarther** [comment]
So I checked this out locally and it's focused on the center for me. Really weird. We should evaluate if its always had weird rendering behavior for you.

**Comment by acmcarther** [comment]
SO yeah this looks good to me, and it works for me. We need to figure out whats up with rendering for you.

:tokyo_tower:
